File: level55/homework/level55.js
function calculate() {
    // მიღებული მნიშვნელობები
    const num1 = parseFloat(document.getElementById("num1").value);
    const num2 = parseFloat(document.getElementById("num2").value);
    const operation = document.getElementById("operation").value;

    let result;
    if (operation === "მიმატება") {
        result = num1 + num2;
    } else if (operation === "გამოკლება") {
        result = num1 - num2;
    } else {
        result = "არასწორი ოპერაცია";
    }

    // შედეგის ჩვენება
    document.getElementById("result").textContent = "შედეგი: " + result;
}

File: level55/homework/level55.py
# davaleba 1
# მომხმარებლისთვის ორი რიცხვის შეყვანა
try:
    num1 = float(input("შეიყვანეთ პირველი რიცხვი: "))
    num2 = float(input("შეიყვანეთ მეორე რიცხვი: "))

    # მომხმარებლისთვის ოპერაციის არჩევა
    operation = input("მიუთითეთ ოპერაცია (მიმატება ან გამოკლება): ").strip()

    if operation == "მიმატება":
        result = num1 + num2
        print(f"შედეგი: {num1} + {num2} = {result}")
    elif operation == "გამოკლება":
        result = num1 - num2
        print(f"შედეგი: {num1} - {num2} = {result}")
    else:
        print("შეყვანილი ოპერაცია არასწორია. სცადეთ 'მიმატება' ან 'გამოკლება'.")
except ValueError:
    print("გთხოვთ, შეიყვანოთ მხოლოდ რიცხვები.")
